README.md

```diff
@@ -1,112 +1,327 @@
+```
+
+Now, we'll just delete the `venv` directory and re-create it.
 
-If you are installing from a toolshed, note that because of how Galaxy uses virtualenv,
-the tool may attempt to use the wrong versions of libraries.
-To fix this, change the line in the `env.sh` that Galaxy creates for the tool that reads something like:
 ```bash
-PYTHONPATH=/home/someuser/shed/irida-galaxy-importer/1.3.0/someuser/irida_export_tool/5d2cb354d0f9/venv/lib/python2.7/site-packages:$PYTHONPATH; export PYTHONPATH 
+rm -rf venv
+
+virtualenv venv
 ```
-to read:
+
+Now, let's activate the virtual environment and re-install the dependencies.
+
 ```bash
-PYTHONPATH=/home/someuser/shed/irida-galaxy-importer/1.3.0/someuser/irida_export_tool/5d2cb354d0f9/venv/lib/python2.7/site-packages
+source venv/bin/activate
+pip2 install -r requirements.txt
 ```
 
+If everything worked, great. We can move on to configuring the tool and Galaxy.
+
+### 2.1.2. Install from GitHub
+
+If, instead, you wish to install the tool via GitHub (e.g., to use the latest code), please follow the instructions below.
+
+*Note: If you've already installed the tool from the ToolShed, you can skip this step.*
+
+#### 2.1.2.1. Clone to Galaxy tools/ directory
 
-#### Prerequisites
+The Galaxy [tools/][galaxy-tools] directory contains tools that come with the Galaxy code. You can add your own to this directory and configure Galaxy to load them up. We will do this for the **IRIDA Import Tool**.
 
-The BioBlend and Requests-OAuthlib libraries are required. The tool will install them if it is being installed from a toolshed.
-They can be manually installed by:
+```bash
+cd galaxy/tools/
+
+git clone -b master https://github.com/phac-nml/irida-galaxy-importer.git
+cd irida-galaxy-importer
 
+# Optional. Checkout specific release from https://github.com/phac-nml/irida-galaxy-importer/releases
+#git checkout [LATEST_RELEASE]
 ```
-pip install bioblend requests-oauthlib
+
+#### 2.1.2.2. Install dependencies
+
+This tool requires Python 2 and a number of Python libraries. You must make sure these are installed and available on all machines this tool will be run with (e.g., if you are submitting to a cluster, these must be available on all cluster nodes).
+
+If you are only running Galaxy on a single machine, please install **Python 2** and use `pip2` to install the dependencies:
+
+```bash
+pip2 install bioblend requests-oauthlib
 ```
 
 You may need to also install the Python and YAML development libraries. On Ubuntu, you can install them with:
 
-```
+```bash
 sudo apt-get install python-dev libyaml-dev
 ```
 
-If you are using Python 2.6, argparse must be installed too. If you are not installing from a toolshed invoke:
+If you are using Python 2.6, `argparse` must be installed too. If you are not installing from a toolshed invoke:
 
-```
-pip install argparse
+```bash
+pip2 install argparse
 ```
 
+#### 2.1.2.3. Configure Galaxy to see tool
```

Comment:
I suggest to move this section to just before `2.3`, as that makes more logical sense to me

Reply:
Thanks Eric. The reason I had this under section 2.1.2 (Install from GitHub) is because modifying the Galaxy `tool_conf.xml` is only really needed if you are not installing the tool from the toolshed.